rime_main.c

```diff
@@ -233,11 +225,20 @@ static void rime_with_ibus() {
 
   gboolean full_check = FALSE;
   ibus_rime_start(full_check);
+
+  // parse & load plugin modules
+  if (!load_plugins("ibus_rime")) {
```

Comment:
nit: don't use `!` to test positive result (unless in shell command).
`if (load_plugins("ibus_rime") == 0)` is more readable.